tests/test_rr1.py

```diff
@@ -0,0 +1,28 @@
+
+import requests
+import time
+import csv
+
+def test_rr1plus_facilities():
+    BASE_URL = "https://uat.directory.spineservices.nhs.uk/ORD/2-0-0/organisations/"
+    errors = {}
+    
+    with open('tables/code_list/rr1plus_facilities.csv', newline='') as csvfile:
```

Comment:
RR1 / RR1+ Codes are issued by the UKRR and so won't match to ODS.

We need an RR1 / RR1+ to ODS mapping, and then we can do such checks. Also some sites are already sending in Coding Standard ODS locations. There are a few further cans of worms with this though.